scripts/verify_opslevel_webhook_signature/go/signature.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"net/http"
+	"sort"
+	"strings"
+)
+
+const (
+	HeaderSignatureCanonical = "X-Opslevel-Signature"
+	HeaderSignature          = "X-OpsLevel-Signature"
+	HeaderTimingCanonical    = "X-Opslevel-Timing"
+	HeaderTiming             = "X-OpsLevel-Timing"
+)
+
+// Search for opslevle signature
```

Comment:
```suggestion
// Search for OpsLevel signature
```

Small spelling correction, with some Capitalization thrown in for good measure :)